python/GafferSceneUI/SceneViewUI.py

```diff
@@ -446,6 +456,34 @@ def __pathFilter() :
 
 		return Gaffer.CompoundPathFilter( [ validPathFilter, camerasPathFilter ] )
 
+	def __dragEnter( self, widget, event ) :
+
+		if not isinstance( event.data, IECore.StringVectorData ) :
+			return False
+
+		if len( event.data ) != 1 :
+			return False
+
+		sets = {}
+		with IECore.IgnoredExceptions( Exception ) :
+			with self.getContext() :
+				sets = GafferScene.SceneAlgo.sets( self.getPlug().node()["in"], ( "__cameras", "__lights" ) )
+
+		if not any(
+			s.value.match( event.data[0] ) & IECore.PathMatcher.Result.ExactMatch
```

Comment:
This should be `GafferScene.PathMatcher` for the maintenance branch

Reply:
Thanks for catching that - fixed in 504726b.